cmd/sebak/cmd/genesis.go

```diff
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	"errors"
+	stderrors "errors"
```

Comment:
Why is it needed if `error` is not imported ?

Reply:
@Geod24 I added it for consistency of code usage.